### ISSUE TITLE
mc (midnight-commander) does not show content  of kopia mounted snap

### ISSUE THREAD

**Issue by mabod** [kopia/kopia#4294]
This is on endeavouros with kopia 0.18.2-1 and midnight commander mc 4.8.32-1. Not sure if this is a kopia topic or an mc topic. I try it here first.

When I mount a snap with kopia like this:

`kopia mount xxxx  /tmp/tmp.TEgiQuQKYh`

I can see all the content with the `ls` command:

```
╰─# ls -l /tmp/tmp.TEgiQuQKYh
insgesamt 5,0G
drwxr-xr-x 1 root root 130M 2023-10-17@12:47 boot
drwxr-xr-x 1 root root 7,6M 2023-10-17@12:59 etc
drwxr-xr-x 1 root root 4,8G 2023-10-17@13:00 home
drwx------ 1 root root  27M 2023-10-17@13:00 root
drwxr-xr-x 1 root root 4,1K 2023-10-17@12:41 usr
```

But mc (midnight commander) is showing nothing in this folder:

![grafik](https://github.com/user-attachments/assets/c201dcdb-df40-4cc7-a6ec-bda4adb3853c)

Why can that be?

**Comment by tomasz-c** [comment]
I have the same on mc 4.8.32 and kopia 0.18.2.
It looks like a kopia problem because on version 0.17 it works to view files in mc on the same version.

**Comment by mabod** [comment]
You are right. It was working before. The kopia update to 0.18.2 seems to be the issue. The last mc update is from early September. That is too long ago.

**Comment by tomasz-c** [comment]
This has been fixed in version 0.21.0. I think this issue can be closed.
